workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/README.md

```diff
@@ -0,0 +1,250 @@
+
+Configure the backend by [configuring the permission policy](https://github.com/backstage/backstage/blob/master/docs/permissions/getting-started.md#test-permission-policy). You can use below policy as example.
+
+```typescript
+import { createBackendModule } from '@backstage/backend-plugin-api';
+import {
+  PolicyDecision,
+  AuthorizeResult,
+} from '@backstage/plugin-permission-common';
+import {
+  PermissionPolicy,
+  PolicyQuery,
+  PolicyQueryUser,
+} from '@backstage/plugin-permission-node';
+import { policyExtensionPoint } from '@backstage/plugin-permission-node/alpha';
+
+class MssvPermissionPolicy implements PermissionPolicy {
+  async handle(
+    request: PolicyQuery,
+    user: PolicyQueryUser,
+  ): Promise<PolicyDecision> {
+    const isGuestUser = user.info.userEntityRef === 'user:development/guest';
+    if (isGuestUser && request.permission.name === 'mssv.view.read') {
+      return { result: AuthorizeResult.DENY };
+    }
+    return { result: AuthorizeResult.ALLOW };
+  }
+}
+
+export default createBackendModule({
+  pluginId: 'permission',
+  moduleId: 'permission-policy',
+  register(reg) {
+    reg.registerInit({
+      deps: { policy: policyExtensionPoint },
+      async init({ policy }) {
+        policy.setPolicy(new MssvPermissionPolicy());
+      },
+    });
+  },
+});
+```
+
+![mssv-rbac](./docs/images/mssv-rbac.png)
+
+#### Procedure
+
+1. Install the plugin using the following command:
+
+```
+yarn workspace app add @backstage-community/multi-source-security-viewer
+```
+
+To enable the PipelineRun list in the CI/CD tab (or any other tab) on the entity view page, add the following snippet in the packages/app/src/components/catalog/EntityPage.tsx.
+
+```
+/* highlight-add-next-line */
+import {
+  isMultiCIAvailable,
+  EntityMultiCIPipelinesContent,
+} from '@backstage-community/plugin-multi-source-security-viewer';
+
+import { EntityJenkinsContent } from '@backstage-community/plugin-jenkins';
+import { EntityGithubActionsContent } from '@backstage-community/plugin-github-actions';
+
+const cicdContent = (
+  <EntitySwitch
+    {/* ... */}
+    {/* highlight-add-start */}
+    <EntitySwitch.Case if={isMultiCIAvailable}>
+      <EntityJenkinsContent />
+      <EntityGithubActionsContent />
+      <EntityMultiCIPipelinesContent />
+      <EntityMultiCIPipelinesContent />
+    </EntitySwitch.Case>
+    {/* highlight-add-end */}
+  </EntitySwitch>
+);
+```
+
+## For users
+
+### Using the plugin in Backstage
+
+#### Prerequisites
+
+1. Your backstage application is installed and running.
+2. You have installed the Multi Source Security Viewer plugin.
+3. Ensure your CI annotations are set:
+
+- [Jenkins](../../../jenkins/plugins/jenkins/README.md)
+- [Github Actions]([../../../github-actions/plugins/github-actions/README.md)
+- [Gitlab CI](https://github.com/immobiliare/backstage-plugin-gitlab?tab=readme-ov-file#annotations)
+
+### Procedure
```

Comment:
Would also update this to use something other than the CI/CD tab.

Reply:
Moved to security